csharp/downgrades/initial/semmlecode.csharp.dbscheme

```diff
@@ -245,7 +245,7 @@ tokens(
               | @destructor | @callable_accessor | @value_or_ref_type | @declaration_with_accessors
               | @local_function;
 
-/** LOCATIONS, ASEMMBLIES, MODULES, FILES and FOLDERS **/
```

Comment:
Even though there are spelling errors we shouldn't change anything in the `dbscheme` files as the hash of these files are used to determine upgrade/downgrade order.